packages/ses/src/inert.js

```diff
@@ -6,6 +6,14 @@ export const InertCompartment = function Compartment(
   throw new TypeError('Not available');
 };
 
+export const InertModularCompartment = function Compartment(
+  _endowments = {},
+  _modules = {},
+  _options = {},
+) {
+  throw new TypeError('Not available');
+};
```

Comment:
There is something a bit strange going on here. 

Even when we're only using the evaluator level by itself, `inert.js' still exports `InertModularCompartment` and `InertStaticModuleRecord`. `get-anonymous-intrinsics.js` still bind these to `%InertModularCompartment%` and `%InertStaticModuleRecord%`, which are distinct names that still appear in the whitelist, even though these are not actually intrinsics that should be visible to user code.

Symmetrically, even when we're running at the module level of abstraction, `inert.js` still exports `InertCompartment` which `get-anonymous-intrinsics.js` still binds to `%InertCompartment%`, even though this object is not actually an intrinsic that should be visible to user code.

But `whitelist.js` shows both constructors`.prototype` pointing at a shared `%CompartmentPrototype%`, which makes sense. However, the whitelist shows `%CompartmentPrototype%.constructor` pointing back at `%InertCompartment%` rather than `%InertModularCompartment%`. No one is ever going to care, because these objects are equivalently inert, but this will be wrong at the module-level of abstraction. The opposite would be wrong at the evaluation-only level of abstraction.

I suggest that the answer is that there should only be one `InertCompartment` constructor which is bound to `%InertCompartment%` and pointed at by `%CompartmentPrototype%`. There should be no whitelist entry for `%InertModularConstructor%`.

Fortunately, both inert constructors are equivalently inert, so using one instead should be easy.